doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -130,14 +130,16 @@ PyArray_Type
 .. c:member:: PyObject *PyArrayObject.base
 
     This member is used to hold a pointer to another Python object that
-    is related to this array. There are two use cases: 1) If this array
-    does not own its own memory, then base points to the Python object
-    that owns it (perhaps another array object), 2) If this array has
-    the (deprecated) :c:data:`NPY_ARRAY_UPDATEIFCOPY` or
-    :c:data:NPY_ARRAY_WRITEBACKIFCOPY`: flag set, then this array is
-    a working copy of a "misbehaved" array. When
-    ``PyArray_ResolveWritebackIfCopy`` is called, the array pointed to by base
-    will be updated with the contents of this array.
+    is related to this array. There are two use cases:
+
+    - If this array does not own its own memory, then base points to the
+    Python object that owns it (perhaps another array object)
```

Comment:
This line should be indented to match the one above. Since I pushed a commit, you need to do
```
git pull kritisingh1 refactor-1  
```
to get my commit before continuing work